src/components/About.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  import Avatar from './Avatar.svelte';
+
+  const avatars = [
+    {
+      name: 'Jérôme Pogeant',
+      img: './images/jerome.jpg',
+      bio: 'jerome_bio',
+      links: {
+        portfolio: 'https://jeromepogeant.com/',
+        github: 'https://github.com/Jerome1337',
+        linkedin: 'https://www.linkedin.com/in/jeromepogeant/',
+        twitter: 'https://twitter.com/jeromepogeant',
+      },
+    },
+    {
+      name: 'Robin Baraffe',
+      img: './images/robin.jpg',
+      bio: 'robin_bio',
+      links: {
+        portfolio: '',
+        github: 'https://github.com/Thyrael',
+        linkedin: 'www.linkedin.com/in/robin-baraffe',
+        twitter: 'https://twitter.com/RobinBaraffe',
+      },
+    },
+  ];
+</script>
+
+{#each avatars as avatar}
```

Comment:
It would be better to add a container to have all these "avatars" inside a given block